shell/platform/windows/text_input_plugin_unittest.cc

```diff
@@ -366,6 +368,60 @@ TEST(TextInputPluginTest, TextEditingWorksWithDeltaModel) {
   // Passes if it did not crash
 }
 
+TEST(TextInputPluginTest, CompositionCursorPos) {
```

Comment:
Consider adding the following comment:
`// Regression test for https://github.com/flutter/flutter/issues/123749`.